src/app/containers/Metadata/index.jsx

```diff
@@ -95,21 +103,23 @@ const MetadataContainer = ({ metadata, promo }) => {
         ampLink={ampLink}
         appleTouchIcon={appleTouchIcon}
         articleAuthor={articleAuthor}
-        articleSection={metadata.passport.genre}
+        articleSection={metadata.passport ? metadata.passport.genre : null}
```

Comment:
I know its not in review yet, but IMO we shouldn't do ternary logic here as it doesn't scale well. Instead we should split out this into multiple containers, 1 called Metadata which sets meta based on the `ServiceContext` values, then 2 separate ones called FrontpageMetadata and ArticleMetadata which handle their own data passing to component.

We can likely have all 3 share 1 or 2 components, just have all metatags in the component dependant on a prop like
```
{ defaultImage && <meta name="og:image" content={defaultImage} /> }
{ articleAuthor && <meta name="article:author" content={articleAuthor} /> }
{ title && <meta property="og:title" content={title}> }
```

In the above example, 
Articles & Frontpage calls the MetaData, which calls the component with `defaultImage`
Articles calls ArticleMetadata, which calls the component with `title` and `articleAuthor`
Frontpage calls FrontpageMetadata, which calls the component with `title` only

In the end, with 3 containers and 1 component, you get all the tags needed for the 2 types of pages, without any need to do ternary logic between 2 data types

Reply:
As mentioned at stand up, initially I considered creating two separate containers - Article and Index, however looking closer into all the metadata tags we need for Index/Front Page, it looks like there are only 5 tags that are only used for articles only. Those are:
* <meta name="article:author" content={articleAuthor} />
* <meta name="article:modified_time" content={timeLastPublished} />
* <meta name="article:published_time" content={timeFirstPublished} />
* <meta name="article:section" content={articleSection} />
* <meta name="article:tag" content={tag} key={tag} />

Imo it doesn't really make sense to create separate containers for this work. The Metadata container can be cleaned up from how it looks now by making sure that the correct values are used depending on the type of the page, but I'd leave it as a single container.

Happy to get input from other devs/pods.

Reply:
I think it'd be more maintainable to keep a single container & component.
We can have a boolean toggle to show these optional fields in the Metadata component:
e.g. `showArticleTags`. If this is true, we show all 5 of these `<meta name="article:*"....>` tags.
Then the props `articleAuthor` `timeLastPublished`, etc can be optional ones. 
This way, we won't need all the ternaries here.

Reply:
To clarify - in this Metadata Container file, we can have `articleProps` constant that handles all of the article props and it's only passed into the Metadata Component if `pagetype === 'article'`. 

This follows on from a conversation @dr3 and I were having about using a `pagetype` value as a toggle - where you can have `pagetype` `'article'` or `'frontpage'`.